project.clj

```diff
@@ -7,8 +7,8 @@
 
   :source-paths ["src/clj" "src/cljs"]
 
-  :dependencies [[org.clojure/clojure "1.6.0" :scope "provided"]
-                 [org.clojure/clojurescript "0.0-2371" :scope "provided"]
+  :dependencies [[org.clojure/clojure "1.8.0" :scope "provided"]
```

Comment:
Do these upgrades need to be done as part of this change? If they're separate it can make it easier to bisect any breakage down the line.